tools/serverpod_cli/lib/src/downloads/resource_manager.dart

```diff
@@ -149,6 +212,20 @@ class ResourceManager {
   }
 }
 
+class ServerpodCloudData {
+  late final String token;
+
+  ServerpodCloudData(this.token);
+
+  factory ServerpodCloudData.fromJson(Map<String, dynamic> json) {
+    return ServerpodCloudData(json['token'] as String);
+  }
+
+  Map<String, dynamic> toJson() => {
+        'token': token,
+      };
```

Comment:
Question: Formatting looks strange here.

Reply:
This is correct since it is an arrow function; the `;` marks the end of the arrow function definition.